src/Resources/contao/forms/FormCaptcha.php

```diff
@@ -188,12 +205,9 @@ protected function getQuestion()
 	 */
 	protected function getSum()
 	{
-		/** @var SessionInterface $objSession */
-		$objSession = \System::getContainer()->get('session');
-
-		$arrCaptcha = $objSession->get('captcha_' . $this->strId);
+		$this->generateCaptcha();
```

Comment:
I would prefer a more explicit check here:

```php
if (!isset($this->arrCaptcha['sum']))
{
    $this->generateCaptcha();
}
```

Technically, this is not necessary of course, however, it would regenerate `$this->arrCaptcha` in case the required property is not set.